README.md

```
@@ -13,6 +13,12 @@ Assuming you have a clone checkout of the https://github.com/EdgeTX/edgetx repos
 % make edgetx-dev
 ```
 
+For local `docker build`/`podman build`, you can pass the architecture explicitly (for example, from host `uname -m`):
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)